_1327/user_management/tests.py

```diff
@@ -102,3 +106,38 @@ def test_impersonate_wrong_url(self):
 
 		response = self.app.post('/hijack/{username}/'.format(username=user.username), user=self.user, expect_errors=True)
 		self.assertEqual(response.status_code, 400)
+
+
+class _1327AuthenticationBackendTests(WebTest):
```

Comment:
I just added this comment to remind myself that I still need to add tests for the university network. I just forgot them ;)